File: 01-input-output/08-seconds-converter/solution.js
import readline from 'readline/promises';
import {stdin as input, stdout as output} from 'process';

const rl = readline.createInterface({
    input, output
});


let userSeconds = await rl.question("Seconds: ");
userSeconds = parseInt(userSeconds);

let hours = parseInt(userSeconds / 3600);
let remainingSeconds = userSeconds % 3600;
let minutes = parseInt(remainingSeconds / 60);
let seconds = remainingSeconds % 60;

console.log(`${userSeconds} seconds = ${hours}H:${minutes}M:${seconds}S`);

rl.close();

File: 01-input-output/08-seconds-converter/solution.c
#include <stdio.h>


int get_seconds(void);

int main(void){
    int user_seconds = get_seconds();

    int hours = user_seconds / 3600;
    int remaining_seconds = user_seconds % 3600;
    int minutes = remaining_seconds / 60;
    int seconds = remaining_seconds % 60;

    printf("%i seconds = %iH:%iM:%iS\n", user_seconds, hours, minutes, seconds);
    return 0;
}


int get_seconds(void){
    printf("Seconds: ");
    int seconds;
    scanf("%i", &seconds);
    return seconds;
}